StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+PaymentMethodAvailability.swift

```diff
@@ -118,7 +118,7 @@ extension PaymentSheet {
                 return [.returnURL, .userSupportsDelayedPaymentMethods]
             case .AUBECSDebit, .cardPresent, .blik, .weChatPay, .grabPay, .FPX, .giropay, .przelewy24, .EPS,
                 .netBanking, .OXXO, .afterpayClearpay, .payPal, .UPI, .boleto, .klarna, .link, .linkInstantDebit,
-                .affirm, .unknown:
+                .affirm, .cashApp, .unknown:
```

Comment:
This isn't what the PR title says - this will disable cashapp for PI+sfu and SI in addition to displaying as a saved PM.

Reply:
Hmmm. thats interesting that is not the behavior I see.. If the setup intent has Cash App it'll still be displayed. I actually don't even see this function being invoked when loading a setup intent.

https://user-images.githubusercontent.com/88012362/213522637-f7a2d770-d663-42ee-952b-d4b7d81b337d.mp4

Reply:
FWIW, we want to disable Cash App for PI+sfu and SI.

Reply:
Going to investigate this more, seems like a bug